### ISSUE TITLE
Chat widget overflows on narrow viewports

### ISSUE THREAD

**Issue by Woebin** [JohnXLivingston/peertube-plugin-livechat#714]
**Describe the bug**
This is a violation of the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/TR/WCAG22) [SC 1.4.10 Reflow (Level AA)](https://www.w3.org/WAI/WCAG22/Understanding/reflow.html).

Content needs to be viewable on viewport widths of as little as 320px without needing to scroll in two dimensions.

The chat widget has a `min-width` CSS value of 400px, which means it doesn't work properly on smaller screens. Some browsers seem to handle this differently - in Chrome, the video player gets squished and the chat widget takes up all available width.
![Embedded video page on narrow display, with Chrome DevTools showing the min-width attribute that's causing issues.](https://github.com/user-attachments/assets/77bb3e34-9f52-4289-b15b-1954fc3bc19d)
(added text reads "Wasted space, video player narrowed" - may be difficult to read in dark display theme due to transparency, apologies)

In other browsers, the chat runs outside the viewport and users are required to scroll horizontally.

**To Reproduce**
1. Go to a page with both video and chat embedded.
2. Use the browser developer tools or resize the window to set a viewport width of 320px.
3. See the video player fail to use all the horizontal space available.

**Expected behavior**
The video player and chat should both extend to the edge of the viewport width.

**Suggested solution**
Ensure that the chat widget can work on lower-width viewports. Simply removing the min-width value seems to be enough to achieve this, although the page it's embedded in still seems to cause some slight issues.
![Embedded widgets with min-width removed, now using more of the viewport width.](https://github.com/user-attachments/assets/6d204d3a-5f80-4ece-9d83-0a7f2276aa85)

**Server:**
 - Peertube version: 7.2.1
 - peertube-plugin-livechat version: 12.0.3

**Desktop:**
 - OS: MacOS 15.5
 - Browser: Google Chrome Version 137.0.7151.120 (Official Build) (arm64)

**Additional context**
This issue was recorded as part of an accessibility audit carried out by [Axess Lab](https://axesslab.com/), contracted by [HowlRound Theatre Commons](https://howlround.com/).
